app/src/main/java/org/fossasia/pslab/fragment/SavedExperiments.java

```diff
@@ -5,10 +5,12 @@
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
+import android.view.Display;
```

Comment:
Nice work @abhinavraj23 ! :) There're slightly few changes need to be done and then we can approve it! This import is unused. Please remove it.